apps/website/docs/dx-cli.md

```diff
@@ -0,0 +1,103 @@
+---
+id: dx-cli
+title: Using the DX CLI
+sidebar_position: 3
+description: Documentation for @pagopa/dx-cli commands and usage.
+---
+
+The `@pagopa/dx-cli` is a command-line tool that helps teams implement PagoPA
+DevEx guidelines consistently and evolve repositories safely.
+
+## Installation
+
+You can invoke the CLI directly via `npx` without installing globally:
+
+```bash
+npx @pagopa/dx-cli --help
```

Comment:
why npx and not pnpx?

Reply:
`npx` is the de-facto standard for one-shot scripts, it is shipped with node and does not requires other package managers or configurations - it can be used with `yarn` and `pnpm`